www/oidc.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+require_once('../includes/init.php');
+require_once __DIR__.'/../optional-dependencies/oidc/vendor/autoload.php';
```

Comment:
It might make sense here to check if `auth_sp_type != oidc` and exit early. Probably before the oidc vendor autoload.